### DISCUSSION TITLE
Style-loader not working with Angular 12

### DISCUSSION THREAD

**Discussion by rishabhgupta3107** [webpack/webpack#14935] [General]
# Bug report

I am using webpack (downloaded implicitly using @angular-devkit#build-angular verison 12.1.4). We have upgraded our application from Angular 9 to Angular 12 recently.
Earlier the webpack version was 4.42.0 And after the upgrade, the webpack version is 5.65.0

Since the upgrade, style-loader has stopped working.
No compilation error as such.
Just that its not working how it is supposed to.
The syntax I am using to import a new css file is: require("style-loader!./file.css").

**Other relevant information:**
Node.js version: 14.18.0

**Comment by alexander-akait** [comment]
Hod do you use style-loader?

**Reply by rishabhgupta3107** [reply]
require("style-loader!./file.css")

Here file.css which we need to override to main style file

**Reply by alexander-akait** [reply]
You need to set `css-loader` if you need CSS

**Reply by rishabhgupta3107** [reply]
I tried that it gives me the following error :

./node_modules/css-loader/dist/cjs.js!./src/app/file.css - Error: Module build failed (from ./node_modules/css-loader/dist/cjs.js):
CssSyntaxError


(1:1) C:\Users\ei11936\Code Base\cpl-client-v2\src\app\file.css Unknown word

**Reply by alexander-akait** [reply]
`CssSyntaxError`, it means you have invalid CSS syntax, i.e. you CSS is not valid

**Reply by rishabhgupta3107** [reply]
This is how file.css looks :

.mat-toolbar-single-row {
    background:  red; 
    color: red;
}
a:hover{
    color: white;
}

One question, how was style-loader working with angular 9 and not with angular 12 with same piece of code?

**Reply by alexander-akait** [reply]
How you run build? Without configuration/examples how you run build/etc I can't say where it the problem
